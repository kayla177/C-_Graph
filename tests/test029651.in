ENTITY BYH7O15H name_BYH7 paper
ENTITY R7ZBO7QJ name_R7ZB author
ENTITY 2B9O5J0Y name_2B9O institution
RELATIONSHIP BYH7O15H published_in R7ZBO7QJ 9.4
PRINT BYH7O15H
HIGHEST
EXIT
