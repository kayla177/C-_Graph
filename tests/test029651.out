success
success
success
success
R7ZBO7QJ
BYH7O15H R7ZBO7QJ 9.4
